src/ElmFormat.hs

```diff
@@ -183,7 +183,7 @@ format elmVersion (inputFile, inputText) =
     case parseModule (inputFile, inputText) of
         Right modu ->
             let
-                (transformed, info) = Transformer.transform modu
+                (transformed, info) = Transformer.transform modu inputFile
```

Comment:
We need to add not only the module name but also the source directory it's from. That's why we're changing `elm-instrument` at all, and this is the starting point